src/fides/api/ops/schemas/connection_configuration/connection_config.py

```diff
@@ -97,13 +108,34 @@ class Config:
         orm_mode = True
 
 
+class ConnectionConfigurationWithSecretsResponse(ConnectionConfigurationResponse):
+    """Describes the returned schema for a ConnectionConfiguration.
+
+    Secrets are included.
+    """
+
+    secrets: Optional[connection_secrets_schemas] = None
```

Comment:
i thought the `secrets` aren't every going to be on the response for security reasons?

Reply:
You are right, this should only be incoming and not in the response back.